autoload/gen_tags.vim

```diff
@@ -260,11 +260,13 @@ function! gen_tags#isblacklist(path) abort
     endif
   endfor
 
-  for l:re in g:gen_tags#blacklist_re
-    if a:path =~ l:re
-      call gen_tags#echo('Found path ' . a:path . ' to be a blacklisted pattern')
-      return 1
-    endif
+  for l:path in [a:path, fnamemodify(a:path, ':p')]
+    for l:re in g:gen_tags#blacklist_re
+      if a:path =~ l:re
```

Comment:
It looks a typo here, should be `l:path`.

But why not just use this:
```vim
let l:abs_path = fnamemodify(a:path, ':p')
for l:re in g:gen_tags#blacklist_re
  if a:path =~ l:re || l:abs_path=~ l:re
   call gen_tags#echo('Found path ' . a:path . ' to be a blacklisted pattern')
   return 1
  endif
endfor
```

Reply:
Commit issue sorry :( there's a typo indeed. `if` is fine too, I just wanted to see which particular path (abs / rel) matched in debug message so I've used `for`.

Reply:
ok, then two if here. better than two for 😆 
```
if a:path =~ l:re
   call gen_tags#echo('Found path ' . a:path . ' to be a blacklisted pattern')
   return 1
endif

if l:abs_path=~ l:re
   call gen_tags#echo('Found path ' . l:abs_path . ' to be a blacklisted pattern')
   return 1
endif
```

Reply:
So be it :) Updated with two `if`s.